packages/host/app/components/operator-mode.gts

```diff
@@ -0,0 +1,98 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { Button } from '@cardstack/boxel-ui';
+import CardCatalogModal from '@cardstack/host/components/card-catalog-modal';
+import CreateCardModal from '@cardstack/host/components/create-card-modal';
+import { restartableTask } from 'ember-concurrency';
+import {
+  chooseCard,
+  catalogEntryRef,
+  createNewCard,
+  baseRealm,
+} from '@cardstack/runtime-common';
+import { CatalogEntry } from 'https://cardstack.com/base/catalog-entry';
+import type LoaderService from '../services/loader-service';
+import { service } from '@ember/service';
+import type * as CardAPI from 'https://cardstack.com/base/card-api';
+import type { ComponentLike } from '@glint/template';
+import { tracked } from '@glimmer/tracking';
+
+interface Signature {
+  Args: {
+    firstCardInStack: ComponentLike;
+  };
+}
+
+export default class OperatorMode extends Component<Signature> {
+  @tracked stack: ComponentLike[];
+  @service declare loaderService: LoaderService;
+  constructor(owner: unknown, args: any) {
+    super(owner, args);
+
+    this.stack = [this.args.firstCardInStack];
+  }
+
+  @action
+  async createNew() {
```

Comment:
Not a blocker, but perhaps this functionality, which should eventually be available for cards to invoke (i.e. "+" button within realm card), may better be encapsulated by an Ember Service.

Reply:
Agree with this as well. Will create/adjust a ticket to reflect that idea.